src/pages/Main/MainEffect/MainEffect.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const MainEffect = () => {
+  return (
+    <div>
+      <Container>
+        {EffectData.map((data, idx) => {
+          return (
+            <EffectContainer key={idx}>
+              <EffectImg src={data.image} />
+              <EffectDescription>{data.description}</EffectDescription>
+            </EffectContainer>
+          );
+        })}
+      </Container>
+    </div>
+  );
+};
+
+const Container = styled.section`
+  position: relative;
+  background: #ffffff;
+`;
+
+const EffectContainer = styled.div`
+  width: 1100px;
+  margin: 0 auto;
+  padding: 100px 20px 0 20px;
+`;
+
+const EffectImg = styled.img`
+  position: relative;
+  width: 100%;
+  margin: 0 auto;
+`;
+
+const EffectDescription = styled.p`
+  margin: 40px -20px 0 -20px;
```

Comment:
아래와 같이 사용하실 수 있습니다. :) 
```suggestion
  margin: 40px -20px 0;
```

Reply:
넵 확인해서 수정했습니다!